lib/NetworkManager.js

```diff
@@ -118,17 +156,19 @@ class NetworkManager extends EventEmitter {
    * @param {!Object} requestPayload
    */
   _handleRequestStart(requestId, interceptionId, url, resourceType, requestPayload) {
-    const request = new Request(this._client, requestId, interceptionId, url, resourceType, requestPayload);
+    const request = new Request(this._client, requestId, interceptionId, this._userRequestInterceptionEnabled, url, resourceType, requestPayload);
     this._requestIdToRequest.set(requestId, request);
     this._interceptionIdToRequest.set(interceptionId, request);
+    if (!this._userRequestInterceptionEnabled && this._protocolRequestInterceptionEnabled)
```

Comment:
Let's move this to _onRequestIntercepted.

Reply:
Done.